src/test/memory-leak/tests/desktopTryItNow.js

```diff
@@ -2,8 +2,13 @@ const { loadEnvConfig } = require('@next/env');
 
 const projectDir = process.cwd();
 loadEnvConfig(projectDir);
+
 function url() {
   return process.env.MEMLAB_WEBSITE_URL;
 }
 
-module.exports = { url };
+async function action(page) {
+  await page.click('a[href="#signUp"]');
```

Comment:
i dont what is this 'a[href="#signUp"]' needs to create const and add the correct name for this, need to make it everywhere